andela/ah-frontend-sparta
# #164798163 User should signup

#### What does this PR do?
Enable user to signup to authors haven

#### Description of Task to be completed?
Provide the user with a signup form to enable them log in to authors haven

#### How should this be manually tested?
- Fetch this branch git fetch origin `ft-user-should-be-able-to-signup-164798163`
- Checkout to the branch git checkout `ft-user-should-be-able-to-signup-164798163`
- Run npm start to view the landing page
- Click on accounts to view the signup and signin pages

#### What are the relevant pivotal tracker stories?
[#164798163](https://www.pivotaltracker.com/story/show/164798163)

#### Screenshots (if appropriate)
<img width="1440" alt="Screenshot 2019-05-07 at 18 05 38" src="https://user-images.githubusercontent.com/35653907/57310658-d2a5f980-70f2-11e9-9885-0c9bea0df00b.png">
<img width="1440" alt="Screenshot 2019-05-07 at 18 09 32" src="https://user-images.githubusercontent.com/35653907/57310907-4f38d800-70f3-11e9-9720-fa676254c09d.png">



#### Team coding conventions
[convention document](https://docs.google.com/document/d/16MmWdTtDQ_IxfAXsRa5n-zS-nvESeTE5ITKX9it1NYQ/edit)
